packages/components/pie-notification/test/visual/pie-notification.spec.ts

```diff
@@ -114,6 +115,31 @@ test.describe('Props', () => {
             });
         });
 
+        test.describe('Action button sizes', () => {
```

Comment:
Just my opinion, but perhaps we should leverage the existing variants story rather than creating a new test? You'll need to add the new `size` prop to the props object in the Story to achieve this - https://webc-testing.pie.design/?path=/story/notification--neutral-prop-variations

Reply:
As discussed, leaving this for now.